radis/lbl/calc.py

```diff
@@ -244,7 +251,7 @@ def calc_spectrum(
     ----------
 ​
     .. [1] RADIS doc: `Spectrum how to? <https://radis.readthedocs.io/en/latest/spectrum/spectrum.html#label-spectrum>`__
-​
+​   .. [2] RADIS GPU support: 'GPU Calculations on RADIS <https://radis.readthedocs.io/en/latest/lbl/gpu.html>'
```

Comment:
Add a direct label (even if the page changes)
```suggestion
​   .. [2] RADIS GPU support: :ref:'GPU Calculations on RADIS <label_gpu_calculations>'
```

In `gpu.rst` add : 

```
.. _label_gpu_calculations: 

# how to make it work + pointers to Issues, GSoC project if needed, etc. 
```

Also make sure `gpu.rst` is imported in `lbl.rst` (for instance under : "Advanced" or "Performance") 

```

.. include:: rst.rst
```